WTWSMS/history/provinces/478 - Monemvasia.txt

```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
 b_monemvasia = castle
 
+b_sparta = city
```

Comment:
It did exist. The city of Sparta has always existed, plus some of my code depends on it.